src/components/Errors/ErrorMessage.js

```diff
@@ -17,7 +17,7 @@ const ErrorMessage = ({children, modifier = 'error'}) => {
   const errorRef = useScrollIntoView();
   if (!children) return null;
   return (
-    <Alert type={modifier} icon={ICONS[modifier]} ref={errorRef}>
+    <Alert type={modifier} icon={ICONS[modifier]} ref={errorRef} role="alert">
```

Comment:
shouldn't we limit this to the warning/error modifiers? I'm not convinced that info/success require the users immediate attention.

Quoting from https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role

> Dynamic changes that are less urgent should use a less aggressive method, such as including `aria-live="polite"` or using an other live region role like [status](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role)

Reply:
Ah check, yeah good one. The role status sounds like a good option for info/success

Reply:
If changed the role to `status` and added a `aria-live="polite"` to the info and ok ErrorMessages, as suggested by https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes